Rakefile

```diff
@@ -1,6 +1,7 @@
 require 'fileutils'
 require 'rspec/core/rake_task'
 require 'tmpdir'
+require 'aws-sdk-resources'
```

Comment:
The dependency is pulled in by `awspec` so I don't _think_ it also needs to be included in the Gemfile.